util/client.py

```diff
@@ -0,0 +1,55 @@
+import urllib
+import os
+import json
+import threading
+from kivy.network.urlrequest import UrlRequest
+
+API_KEY = ' '
+
+# KEY REQUEST
+
+
+# read and parse key.json
+def get_options():
+	util = os.path.dirname(os.path.abspath(__file__))
+	key_json = os.path.join(util, 'key.json')
+	with open(key_json) as o:
+		data = json.load(o)
+	return data
+
+# main authorize function
```

Comment:
do you recommend a certain time? I just changed it to a hour but maybe a day?

Reply:
Why do we need to reauthorize?

Reply:
We need to reauthorize depending on the Auth0 token refresh cycle. But since we authorize when they login, it probably is not necessary since its very unlikely they will be using the game long enough to expire the token.

Reply:
Let's talk about this on slack. I think we need more thought in when they need wifi